<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMemoryInfo</class>
 <widget class="QWidget" name="FormMemoryInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelFontMemory">
           <property name="text">
            <string>Memory font</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPositionMemory">
           <property name="text">
            <string>Position</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="FontButton" name="toolButtonFontMemory">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxPositionMemory">
           <item>
            <property name="text">
             <string>Top</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Middle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bottom</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxFullScreenMemory">
       <property name="text">
        <string>Full screen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxShowRythmTextMemory">
       <property name="text">
        <string>Show rythm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelExample">
       <property name="text">
        <string>Example </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FontButton</class>
   <extends>QToolButton</extends>
   <header>fontbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
